src/Microsoft.Health.Dicom.Api/Registration/Swagger/ConfigureSwaggerOptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Microsoft.Health.Dicom.Api.Registration.Swagger
+{
+    public class ConfigureSwaggerOptions : IConfigureOptions<SwaggerGenOptions>
+    {
+        private readonly IApiVersionDescriptionProvider _provider;
+
+        public ConfigureSwaggerOptions(IApiVersionDescriptionProvider provider) =>
+          _provider = provider;
+
+        public void Configure(SwaggerGenOptions options)
+        {
+            foreach (var description in _provider.ApiVersionDescriptions)
+            {
+                options.SwaggerDoc(
+                  description.GroupName,
+                    new OpenApiInfo()
+                    {
+                        Title = $"API {description.ApiVersion}",
```

Comment:
I don't think we need version in the title as there's a separate field for version. So perhaps we should call this something more formal like `"Medical Imaging Server for DICOM"`? We should also fill out the other fields like description, contact, license, etc

Reply:
I updated the title and added the license. I didn't know what to put for contact or description so I havent added them, but if you have any ideas I can add them!